src/coreclr/debug/daccess/daccess.cpp

```diff
@@ -4981,14 +4981,15 @@ ClrDataAccess::SetCodeNotifications(
                 {
                     for (ULONG32 check = 0; check < numTokens; check++)
                     {
-                        if (!IsValidMethodCodeNotification(flags[check]))
+                        _ASSERTE(flags[check] <= USHRT_MAX);
```

Comment:
This should not be needed. IsValidMethodCodeNotification was changed to take ULONG